src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class MainApp {
+    public static void main(String[] args) throws IOException {
```

Comment:
don't see point of adding IOEception to main method signature, it's better to wrap code that throws this exception into try catch block and if use enter invalid file, notify him about that and ask to re-enter path

Reply:
done. but not sure about my decision about try-catch wrapping